src/get-jwks.d.ts

```diff
@@ -13,7 +13,7 @@ type GetPublicKeyOptions = {
 type GetJwksOptions = {
   max?: number
   ttl?: number
-  allowedDomains?: string[]
+  issuersWhitelist?: string[]
```

Comment:
This might be a breaking change if it's used in a typescript project.

Perhaps, we can still include `allowedDomains` rather than replacing it?

Reply:
ah good catch yeah. I think that as long as this causes a build breakage I'm happy to do this change without bumping a major version